js-react/app.html

```diff
@@ -6,6 +6,7 @@
     <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
     <script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
   </head>
-  <div id="container"></div>
+  <script type="text/javascript" src="newtonraphson.js"></script>
```

Comment:
Web assembly files are not in cwd so live demo fails, please add newtonraphson.js and newtonraphson.wasm to dir.

Reply:
They are added. So it should work now.

Reply:
Thanks, yep now it works.